instrumentation/kamon-spring/src/main/scala/kamon/instrumentation/spring/client/ClientInstrumentation.scala

```diff
@@ -0,0 +1,106 @@
+package kamon.instrumentation.spring.client
+
+import com.typesafe.config.ConfigFactory
+import kamon.Kamon
+import kamon.instrumentation.http.HttpClientInstrumentation.RequestHandler
+import kamon.instrumentation.http.{HttpClientInstrumentation, HttpMessage}
+import org.springframework.web.reactive.function.client.{ClientRequest, ClientResponse}
+import reactor.core.publisher.Mono
+
+import java.util.function.Consumer
+import scala.collection.JavaConverters.mapAsScalaMapConverter
+import scala.collection.mutable
+
+object ClientInstrumentation {
+  private val instrumentation: HttpClientInstrumentation =
+    HttpClientInstrumentation.from(ConfigFactory.empty(), "spring.client")
```

Comment:
Why an empty config here? I think we should do something similar to what we do with all other instrumentations: have a dedicated path in the config for this specific implementation and fallback to the defaults.